ReactAndroid/src/main/java/com/facebook/react/uimanager/TouchTargetHelper.java

```diff
@@ -200,16 +200,10 @@ private static boolean isTransformedTouchPointInView(
       float eventCoords[], View view) {
     PointerEvents pointerEvents = view instanceof ReactPointerEventsView ?
         ((ReactPointerEventsView) view).getPointerEvents() : PointerEvents.AUTO;
-    if (pointerEvents == PointerEvents.NONE) {
-      // This view and its children can't be the target
-      return null;
-
-    } else if (pointerEvents == PointerEvents.BOX_ONLY) {
-      // This view is the target, its children don't matter
-      return view;
-
-    } else if (pointerEvents == PointerEvents.BOX_NONE) {
-      // This view can't be the target, but its children might
+    if (!view.isEnabled() || pointerEvents == PointerEvents.BOX_NONE) {
```

Comment:
It seems like this would no longer respect PointerEvents.NONE/BOX_ONLY if the view is disabled. I think you need to explicitly handle the disabled case in each of the places we might return `view` as the target (e.g. BOX_ONLY and AUTO